res/values/cr_strings.xml

```diff
@@ -2524,4 +2524,8 @@
 
     <!-- Sound: Title for the option defining the phone ringtone for slot2 on Muilti SIM device. [CHAR LIMIT=50] -->
     <string name="ringtone2_title">Phone ringtone - SIM 2</string>
+
+    <!-- Fingerprint unlock without bouncer -->
+    <string name="unlock_without_bouncer_title">Fingerprint unlock on notification double tap</string>
+    <string name="unlock_without_bouncer_summary">Unlock device with fingerprint on notification double tap without pin/password/pattern</string>
```

Comment:
and here add "without need to input"
"Unlocks device with fingerprint on notification double tap without need to input pin/password/pattern"